modules/oc-mirror-migration-differences.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * oc-mirror-migration-v1-to-v2.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oc-mirror-migration-differences_{context}"]
+= Differences between oc-mirror plugin v1 and v2
+
+Before migrating from oc-mirror plugin v1 to v2, you can check differences between oc-mirror plugin v1 to v2. Review the following:
+
+. *Explicit version selection*: Users must explicitly specify `--v2` when using `oc-mirror`. If no version is specified, v1 is executed by default. This behavior is expected to change in future releases (4.19 or later), where `--v2` will be the default.
+. *Command line changes*:   
+- `mirror` to `mirror`: Introduces `--workspace` in the command line.
+- `mirror to disk` & `disk to mirror`: Requires `--v2` and mandates `file://` for disk locations.
+
+. *API version update*: The `ImageSetConfiguration` API version changes from `v1alpha2` (v1) to `v2alpha1` (v2). You must manually update their configuration files before migration.
+. *Configuration changes*: 
+- `storageConfig` must be removed in oc-mirror plugin v2.
+- Incremental mirroring is now handled automatically through the working directory/local cache.
+. *Changes in output paths*: Verify mirrored content in the new paths after migration.
+- Outputs in oc-mirror plugin v2 are not stored in the same location as oc-mirror plugin v1.
+- You must check the cluster-resources folder under the working directory for:
+** IDMS (`ImageDigestMirrorSet`)
+** ITMS (`ImageTagMirrorSet`)
+** `CatalogSource`
+** `ClusterCatalog`
+
+. *Workspace and directory naming*: Follow this to prevent any potential data inconsistencies when transitioning between versions.
+- The v1 `oc-mirror-workspace` is no longer needed.
+- Use a new directory for oc-mirror plugin v2 to avoid conflicts.
+
+. *Replacing ICSP with IDMS/ITMS*:
+- In oc-mirror plugin v2, `ImageContentSourcePolicy` (ICSP) is replaced by `ImageDigestMirrorSet` (IDMS) and `ImageTagMirrorSet` (ITMS).
+- Remove existing ICSPs before applying IDMS/ITMS.
```

Comment:
hi @kasturinarra , @zhouying7780 
Can you help us confirm it is safe to remove ICSP prior to applying IDMS? or if we need to instruct the user to first apply IDMS, ITMS then remove ICSP?

Reply:
ICSP, IDMS and ITMS are different resource , so no necessary to remove ICSP prior to apply the IDMS, but if we run the delete action , we must remove the ICSP.